### DISCUSSION TITLE
How to setting the style of parting line between pannels

### DISCUSSION THREAD

**Discussion by detailyang** [wezterm/wezterm#4620] [Q&A]
### What Operating System(s) are you running on?

macOS

### WezTerm version

20230712-072601-f4abf8fd

### Ask your question!

What's the config to setting the line between pannels as the following:

![image](https://github.com/wez/wezterm/assets/3370345/1a10a6b7-c561-4bae-ae85-d37c6de67fe1)

**Comment by nekowinston** [comment]
See [Defining your own colors](https://wezfurlong.org/wezterm/config/appearance.html#defining-your-own-colors); it's `split`, e.g.

```lua
config.colors = { split = "#ff69b4" }
```
